source/CardGames.py

```diff
@@ -77,6 +81,20 @@ def getCard(self):
     self.discarded.append(card)
     return card
 
+  def drawCard(self):
+      """
+      Draw a card from the top of the deck.
+      Returns:
+        Card: The card drawn from the top of the deck.
+      """
+      if self.size > 0:
+        card = self.getCard()
+        print(f"Card drawn: {card.suit} {card.value}")
+        return card
+      else:
+        print("No cards left in the deck.")
+        return None
+
```

Comment:
This looks good. Now, all we need to do is put this option on the starter screen and figure out a way for users to call this function. Maybe we can bind it to a key somehow, or make it to where they can simply call it in the terminal (without the class.drawCard()).